### PR TITLE
Properly adds per_page parameter

### LINKED ISSUE #538: setPerPage doesn't work in some cases
This doesn't work: 
```
$api = $client->api('users');
$api->setPerPage(100);
$response = $api->all($since);
```
The reason is because the api adds a "?since=1234" to the path and then another "?per_page=100" is added to the path to make an invalid url. The second ? should be an &.

Here is the solution:
AbstractAPI.php::get()

```
...
if (count($parameters) > 0) {
            if(strpos($path,"?")===false) {
                $path .= '?'.http_build_query($parameters);
            } else {
                $path .= '&'.http_build_query($parameters);
            }
}
...
```

Pull request incomming

### PR DESCRIPTION
**Pull request by offsky**
Properly adds per_page parameter to path even if a previous parameter has already been added

Fixes #538

### CODE REVIEW

File: lib/Github/Api/AbstractApi.php [hunk 1]
```diff
@@ -75,7 +75,11 @@ protected function get($path, array $parameters = array(), array $requestHeaders
         }
 
         if (count($parameters) > 0) {
-            $path .= '?'.http_build_query($parameters);
+            if(strpos($path,"?")===false) {
```
Comment (j0k3r) [inline lib/Github/Api/AbstractApi.php]: I think you should put some space here

```php
if (strpos($path, "?") === false) {
```

**Comment by offsky** [comment]
Apparently your style checker doesn't like the spaces you requested. What do you want me to do?

**Comment by j0k3r** [comment]
Mostly because you didn't copy/paste my snippet. You are missing a space between `$path,`& `"?"`

**Comment by offsky** [comment]
Got it. Sorry

**Comment by Nyholm** [comment]
Thank you for this PR. I see that it solves the issue but I do think it is the wrong approach. 

See https://github.com/KnpLabs/php-github-api/issues/538#issuecomment-288649669

**Pull request closed without merging**